script/validate-vet

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+source "$(dirname "$BASH_SOURCE")/.validate"
+
+# # We will eventually get to the point where packages should be the complete list
+# IFS=$'\n'
+# files=( $(validate_diff --diff-filter=ACMR --name-only -- '*.go' | grep -v '^Godeps/' || true) )
+# unset IFS
+packages=(
+    app/cli/*.go
+    version/*.go
```

Comment:
I don't really like the way I've set this up but in the end we'll remove this part and uncomment what's just above :wink: